bitnami/phabricator/values.yaml

```diff
@@ -189,37 +396,65 @@ ingress:
   ##
   enabled: false
 
+  ## Enable TLS configuration for the hostname defined at ingress.hostname parameter
```

Comment:
Why is the ingress block inside the service block? Shouldn't it have its own block for itself? See Magento example

Reply:
The ingress block is not inside the service block. Please double-check

Reply:
I see, probably I was confused with the indentation or something. Nevermind then.